test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -1092,4 +1102,104 @@ public void forceRemoveAfterSeal() throws Exception {
             shutdownCorfuServer(corfuServer);
         }
     }
+
+    /**
+     * Verify that a stream can be correctly loaded after a reconfiguration and
+     * all the updates on this stream are subsumed by a checkpoint and trimmed.
+     */
+    @Test
+    public void testFailoverWithStreamSubsumedByCheckpoint() throws Exception {
```

Comment:
Should also verify that the response for LOG_ADDRESS_SPACE_REQUEST on all servers is equal.

I think we need a more general test case, but this might not be the best place to write it.